x-pack/filebeat/module/sonicwall/firewall/test/general.log-expected.json

```diff
@@ -0,0 +1,639 @@
+[
+    {
+        "event.code": "98",
+        "event.dataset": "sonicwall.firewall",
+        "event.module": "sonicwall",
+        "event.original": "Jan  3 13:45:36 192.168.5.1 id=firewall sn=000SERIAL time=\"2007-01-03 14:48:06\" fw=1.1.1.1 pri=6 c=262144 m=98 msg=\"Connection Opened\" n=23419 src=2.2.2.2:36701:WAN dst=1.1.1.1:50000:WAN proto=tcp/50000",
+        "fileset.name": "firewall",
+        "input.type": "log",
+        "log.flags": [
+            "dissect_parsing_error"
```

Comment:
is the dissect failing?  seems odd we aren't getting src & dst fields out of this.

Reply:
fixed